### ISSUE TITLE
size error with default model.pkl

### ISSUE THREAD

**Issue by akchain** [Aditya-401/Cloth_Size_predictor#1]
hello, I tried weight 60, age 18, height 166 and got predicted size XXXL. Is it a bug? Thanks.

**Comment by Aditya-401** [comment]
Yeah, I'm new to machine learning, either I didn't optimize the model correctly or their is something wrong with the dataset cuz in kagle max accuracy was about 60% with all the eda done.
